### PR TITLE
ads/register payment

### PR DESCRIPTION
**Pull request by dabbeg**
- Updating kennitala package as it now includes ts types
- Ads/adding column to track the date when financial_state is updated
- Ads/Updating financial_state_updated when a flight is cancelled

# \<Description\>

## What

Add financial state updated column

## Why

To be able to know when the state changed for reporting reasons

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

**Review by darrikonn — DISMISSED** [review]

**Review by darrikonn — APPROVED** [review]

File: apps/air-discount-scheme/backend/migrations/20201001142253-fix-financial-state-default-value.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        ALTER TABLE flight_leg ALTER COLUMN financial_state SET DEFAULT 'AWAITING_DEBIT';
```
Comment (sindrig) [inline apps/air-discount-scheme/backend/migrations/20201001142253-fix-financial-state-default-value.js:8]: do you also need to `update table flight_leg set financial_state='AWAITING_DEBIT' where financial_state='awaitingDebig';`?

Comment (darrikonn) [reply apps/air-discount-scheme/backend/migrations/20201001142253-fix-financial-state-default-value.js:8]: Nope.. no records for it -> thankfully handled in the app layer.

However, it doesn't hurt to be extra careful and add it.

Comment (dabbeg) [reply apps/air-discount-scheme/backend/migrations/20201001142253-fix-financial-state-default-value.js:8]: No we checked with the lambda thing, there were no records with `awaitingDebit`

Comment (sindrig) [reply apps/air-discount-scheme/backend/migrations/20201001142253-fix-financial-state-default-value.js:8]: :hindu_temple: :church:

**Pull request merged**